recipes/myst-parser/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "myst-parser" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "70c15370f1d53aa0090efdf22ce1c6be810ce22806251cd0481b59a4657004e6"
+
+build:
+  number: 0
+  entry_points:
+    - myst-benchmark = myst_parser.cli.benchmark:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - docutils >=0.14
+    - mistletoe-ebp ~=0.8
```

Comment:
I don't know that `~=` is valid in conda

Reply:
It's not mentioned here: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html, but at the same time doesn't through any sort of error. I think it's covered here though: https://github.com/conda/conda/blob/master/conda/models/version.py#L397-L416

Reply:
I don't think we should use undocumented features of conda

Reply:
So what you’re are saying is that your going to go and document it then 😉

Reply:
I can change it if you want though? I’m not that fussed

Reply:
The docs do say that they are matched using PEP-440,
which this is part of: https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
If you want to try to get a PR to document it through, that would be amazing!  I am not a big conda/conda-build contributor. It would be easier to get your PR in, though, if you changed it here

Reply:
done